index.html

```diff
@@ -235,24 +241,30 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
       <form id="contact-form" action="https://formspree.io/f/mrgdolqb" method="POST">
         <input type="text" autocomplete="off" name="name" id="name" placeholder="Full name" maxlength="30" required />
 
+        <input type="text" autocomplete="off" name="firstname" id="firstname" placeholder="First name" maxlength="30" required />
+
+        <input type="text" autocomplete="off" name="lastname" id="lastname" placeholder="Last name" maxlength="30" required />
+
```

Comment:
- [ ] Your form is very similar to the Figma template, but there is an important note in the project requirement mentioning that the part of the name input fields should be ignored in the desktop version for templates that has 2 input fields, one for the first name and other for the last name, and use only one input field for the full name in both versions(desktop and mobile). Please remove the input fields you added and use only one input field for the full name as the project requirement asks for.
 - 
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/61919617/201159979-38abda83-fa33-4341-b9c1-f5398e69d037.png">

Reply:
@leonardodiasb thanks, I completely forgot that remark.